.env.example

```diff
@@ -0,0 +1,7 @@
+REACT_APP_PRIVY_APP_ID=clx07qnbf07qdwlja56mg55er
```

Comment:
Just checking here, are those "examples"? or they are really supposed to be used?